tests/php/validation/test-class-amp-validation-error-taxonomy.php

```diff
@@ -994,6 +975,9 @@ public function test_add_admin_notices() {
 	 * @covers \AMP_Validation_Error_Taxonomy::filter_tag_row_actions()
 	 */
 	public function test_filter_tag_row_actions() {
+		wp_set_current_user( $this->factory()->user->create( [ 'role' => 'administrator' ] ) );
+		global $pagenow;
+		$pagenow = 'edit-tags.php';
```

Comment:
I'm confused as to why this is needed now. Perhaps global state had set it before.

Reply:
Yes, this seems to be the case.